### ISSUE TITLE
Add tooltips to the category options, so it's clearer what variables go in what category (i.e. Beverages in the Foods category)

### ISSUE THREAD

**Issue by AMelhede** [curedao/curedao-monorepo#38]
It would be highly beneficial to add a beverage of drinks option as a category

![image](https://user-images.githubusercontent.com/95473109/226520194-e001fd1d-2f87-441e-afdc-280a4a21a327.png)

**Comment by AMelhede** [comment]
Which category do you choose when inputting nutrients like salt, supplements, and other non-food and beverage options?

E.g. Himalayan salt:
![image](https://user-images.githubusercontent.com/95473109/226525313-5fe227f7-7d1f-4c85-8311-2bcf25a5762d.png)

**Comment by mikepsinn** [comment]
Thanks, @AMelhede !  Himalayan Salt would be in the `Foods` category. Supplements would go in the `Treatments` category. How can we make that less confusing for people?

It seems like too many categories could also become a problem if there's a long list, and it's questionable which one something should go in.   Like all, Supplements could also be considered Treatments.  

The easiest way to deal with the Drinks issue, may be for us to add a `displayName` property set to `Foods and Drinks` on the `variableCategory` object when they're fetched. The reset of the categories could just have the same `displayName` as the name. 

Then we can use the `displayName` property on all the variable category selectors.

**Comment by AMelhede** [comment]
Yes, that would be good. Thank you, @mikepsinn!
If we highlight how, salt, spices, herbs, foods, etc. differ but are inputted correctly.
Adding an explanation when holding the cursor above each option, e.g. food, treatment, goals, etc. would be very helpful and could likely eradicate this confusion.
I imagine many people wouldn't be sure how to differ from these seemingly different options, especially, when there is no specific explanation.

**Comment by mikepsinn** [comment]
@AMelhede Yeah, a tooltip popup text explaining what's included in each category when hovering over the different options is a great idea!  If you have any ideas, feel free to write a sentence for each category that you think needs one in a way that's easily understandable for the average user.  Then I or someone can implement it (or I can show you or anyone how if they're interested).

**Comment by AMelhede** [comment]
Awesome. So, do we change any of the current options, as well, or just put all nutrition in the same one, beverages, food, spices, herbs, condiments, etc?
Even just changing it to nutrition instead of food could help a lot, @mikepsinn
